django/db/models/base.py

```diff
@@ -1207,6 +1207,7 @@ def check(cls, **kwargs):
                 *cls._check_index_together(),
                 *cls._check_unique_together(),
                 *cls._check_ordering(),
+                *cls._check_indexes(),
```

Comment:
I'd put this after `_check_unique_together` here, so it's grouped with the other callers of `_check_local_fields`